app/webpacker/scripts/autocomplete.js

```diff
@@ -43,7 +43,7 @@ const initAutocomplete = ($el, $input) => {
         name: input.name,
         defaultValue: input.value,
         minLength: 3,
-        source: request("/provider-suggestions"),
+        source: request(input.getAttribute("data-url")),
```

Comment:
could this be open to xss? Just thinking if the attribute is replaced with something malicious then the lookup could be use as a keylogger.

Reply:
Path is now being passed as param to initAutocomplete function